test/e2e/e2eutil.go

```diff
@@ -42,7 +42,7 @@ func UpdateMongoDBResource(original *mdbv1.MongoDBCommunity, updateFunc func(*md
 	return TestClient.Update(context.TODO(), original)
 }
 
-func NewTestMongoDB(ctx *Context, name string, namespace string) (mdbv1.MongoDBCommunity, mdbv1.MongoDBUser) {
+func NewTestMongoDBArbiter(ctx *Context, name string, namespace string, noMembers int, arbiters int) (mdbv1.MongoDBCommunity, mdbv1.MongoDBUser) {
```

Comment:
Thanks! I would either rename it "NewTestMongoDBWithArbiter" and let it as an exported function that we assume may be useful later on if we need to specify a specific number of members or arbiters.
 If we assume it will not be useful, then just making a private function directly in the test file will be cleaner. 
Another advantage here by having the function in that file is that I don't have very similar code between NewTestMongoDB and NewTestMongoDBArbiter as the former calls the latter

Reply:
we can still call NewTestMongoDB in the arbiter test file, I will leave the choice up to you, I think both are valid!